crowd_anki/export/anki_exporter.py

```diff
@@ -67,3 +71,81 @@ def _copy_media(self, deck, deck_directory):
                             str(media_directory.resolve()))
             except IOError as ioerror:
                 print("Failed to copy a file {}. Full error: {}".format(file_src, ioerror))
+
+    def sort_notes(self, notes):
```

Comment:
I think this function has grown complex enough to deserve it's own class ;) That'd also simplify testing I believe

Reply:
Heh, perhaps! Everything else is just an unused MagicMock, so testing is already simple enough. But yes, a separate class would be a lot neater.